samples/ConsoleSample.CSharp/Program.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using ActiveLogin.Identity.Swedish;
 using ActiveLogin.Identity.Swedish.Extensions;
+using ActiveLogin.Identity.Swedish.TestData;
 
 namespace ConsoleSample
```

Comment:
Could we have the .IsTestData() extension in the other sample as well? The one that shows the hints etc.

Reply:
Do you mean add it to the output of this method?
```csharp
private static void WritePersonalIdentityNumberInfo(SwedishPersonalIdentityNumber personalIdentityNumber) 
{
    ...
    WriteKeyValueInfo("   .IsTestNumber()", personalIdentityNumber.IsTestNumber().ToString());
}
```

Reply:
Do you mean add it in this method:

```csharp
private static void WritePersonalIdentityNumberInfo(SwedishPersonalIdentityNumber personalIdentityNumber)
{
    ...
    WriteKeyValueInfo("   .IsTestNumber()", personalIdentityNumber.IsTestNumber().ToString());
}
```

Reply:
Yes, that was what I was thinking of. Do you think it makes sense?
Maybe a one line comment above that line to specify that it requires the testdata package.

Reply:
Ok, this is fixed.